test/controllers/DeleteAdditionalMetadataControllerSpec.scala

```diff
@@ -0,0 +1,226 @@
+package controllers
+
+import cats.implicits.catsSyntaxOptionId
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.client.WireMock.{containing, okJson, post, postRequestedFor, urlEqualTo}
+import configuration.GraphQLConfiguration
+import graphql.codegen.GetConsignment.getConsignment
+import io.circe.generic.auto.exportEncoder
+import io.circe.syntax.EncoderOps
+import org.scalatest.concurrent.ScalaFutures.convertScalaFuture
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{GET, contentAsString, contentType, defaultAwaitTimeout, redirectLocation, status}
+import services.{ConsignmentService, CustomMetadataService}
+import testUtils.{CheckPageForStaticElements, FrontEndTestHelper}
+import uk.gov.nationalarchives.tdr.GraphQLClient.Error
+import play.api.http.Status.{FORBIDDEN, FOUND, OK, SEE_OTHER}
+
+import java.util.UUID
+import scala.concurrent.ExecutionContext
+
+class DeleteAdditionalMetadataControllerSpec extends FrontEndTestHelper {
+
+  val wiremockServer = new WireMockServer(9006)
+
+  override def beforeEach(): Unit = {
+    wiremockServer.start()
+  }
+
+  override def afterEach(): Unit = {
+    wiremockServer.resetAll()
+    wiremockServer.stop()
+  }
+
+  val checkPageForStaticElements = new CheckPageForStaticElements
+  implicit val ec: ExecutionContext = ExecutionContext.global
+
+  val fileIds: List[UUID] = List(UUID.randomUUID())
+
+  "confirmDeleteAdditionalMetadata" should {
+    "render the delete closure metadata page with an authenticated user" in {
```

Comment:
Could you also add a test for an unauthenticated user?

Reply:
We already have one test for an unauthenticated user `redirect to the login page if the page is accessed by a logged out user`. It says logged out user but it is the same.